freeipa/src/main/java/com/sequenceiq/freeipa/service/stack/StackService.java

```diff
@@ -109,6 +109,14 @@ public Stack getByCrnAndAccountIdEvenIfTerminated(String environmentCrn, String
         return stacks;
     }
 
+    public List<Stack> getMultipleByEnvironmentCrnAndAccountIdEvenIfTerminatedWithList(String environmentCrn, String accountId) {
+        List<Stack> stacks = findMultipleByEnvironmentCrnAndAccountIdEvenIfTerminatedWithList(environmentCrn, accountId);
+        if (stacks.isEmpty()) {
+            throw new NotFoundException(String.format("FreeIPA stack by environment [%s] not found", environmentCrn));
```

Comment:
I would expect it would return an empty list and not throw anything

Reply:
The "find" APIs seem to allow empty lists while the "get" APIs seem to require contents. Lets also discuss this. Maybe I am misunderstanding the intent.

Reply:
that's right for single results. If you check the `StackService` in core, all the calls which return a collection never checks if the result is empty, it's just returns an empty result. I think it's better for user experience too. Like on UI they could just display an empty table if the result is empty, but if an error is thrown they would need extra coding there.

Reply:
I removed the "get" methods that return `List<Stack>` and now the "find" versions are used. As you recommended this no longer throws exceptions but rather returns an empty list.